src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/InterceptorConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.overseasentitiesapi.configuration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.lang.NonNull;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.companieshouse.overseasentitiesapi.interceptor.TransactionInterceptor;
+
+@Configuration
+public class InterceptorConfig implements WebMvcConfigurer {
```

Comment:
Are you missing a 'component scan' annotation here?

Reply:
don't think so, the scan in confirmation-statement was because we were importing an interceptor from a pom dependency that had a different package structure, so we had to use ComponentScan and specify it's path to pick it up. We'll have to add that interceptor at some point as it is security role related. It's working as @Configuartion is also a @Component so can see other beans in the project.